git-missing: add page (#6126)
tldr-pages/tldr: pages/common/git-missing.md

# git missing

> > Show commits which aren't shared between 2 branches.

Reviewer: ```suggestion
> Show commits which aren't shared between two branches.
```

> > Show commits which aren't shared between two branches.

Reviewer 2: ```suggestion
> Show commits which are not shared between two branches.
```

`aren't` does not feel formal

Author: aren't is more than acceptable, in fact I would argue `are not` is incorrect.
Also although its only a few, `are not` is used 8 files while `aren't` has 11 files

Reviewer 2: Should we be consistent and change are not to aren't in those pages as well ?

Author: Whats wrong with aren't, its better, reads nicer

Reviewer: It's merely a matter of preference.  I personally prefer `are not` here because of the formality (`aren't` sounds just a tad too long and informal to me).

>in fact I would argue `are not` is incorrect.

_That_ is incorrect — `aren't` is the contraction of `are` and `not`; thus, `are not` and `aren't` are identical.

Reviewer: >Should we be consistent and change are not to aren't in those pages as well ?

It's not really something that ought to be standardized.  I don't think we should do that.

>Also although its only a few, are not is used 8 files while aren't has 11 files

Those numbers are too small to be meaningful :)

Author: aren't definitely reads shorter than are not

Reviewer 2: Yes I agree, it's merely preference. I think this is a nitpick.
Personally this felt informal, but looks like more files have 'aren't'.

Reviewer: >aren't definitely reads shorter than are not

You're right — the was I was pronouncing it in my head sound longer (`arnt` vs `arent`) 🙂

Reviewer: >Personally this felt informal, but looks like more files have 'aren't'.

As I said before, those numbers are _so_ small (and similar) that it's not a good idea to base decisions of them.

Author: so what are we doing here

Reviewer: Let's keep it as `aren't`, because I don't really see why we should change it.

> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-missing>.

> - Show commits which aren't shared between the current checked-out branch and a specified branch:

Reviewer: ```suggestion
- Show commits which aren't shared between the currently checked-out branch and another branch:
```

> - Show commits which aren't shared between the currently checked-out branch and another branch:

Reviewer 2: ```suggestion
- Show commits which are not shared between the currently checked-out branch and another branch:
```

`git missing {{branch}}`

> - Show commits which aren't shared between two branches:

Reviewer 2: ```suggestion
- Show commits which are not shared between two branches:
```

> - Show commits which aren't shared between 2 specified branches:

Reviewer: ```suggestion
- Show commits which aren't shared between two branches:
```
